code/modules/chemistry/Reagents-Diseases.dm

```diff
@@ -90,6 +90,20 @@ datum
 			transparency = 235
 			disease = /datum/ailment/disease/cold
 
+
+		disease/lungrot
+			name = "lungrot bloom"
+			id = "lungrot_bloom"
+			description = "highly toxic fungal colonies created in the enviroment of a weakened lung."
+			reagent_state = SOLID
+			minimum_to_infect = 2.5
```

Comment:
Considering the likelyhood of walking into a random miasma cloud while using salbutamol, either this number should be much higher or the amount produced should be much lower.